src/Uno.UWP/Generated/3.0.0.0/Windows.Devices.Geolocation/Geolocator.cs

```diff
@@ -21,7 +21,7 @@ public  uint ReportInterval
 			}
 		}
 		#endif
-		#if __ANDROID__ || __IOS__ || NET461 || __WASM__ || __MACOS__
+		#if false || __IOS__ || NET461 || __WASM__ || __MACOS__
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4826530)